docs/source/devguide/workflow.rst

```diff
@@ -67,6 +67,11 @@ features and bug fixes. The general steps for contributing are as follows:
        cd openmc
        git checkout -b newbranch develop
 
+3. Run ``tools/dev/install-commit-hooks.sh`` to install a post-commit hook that
+   runs clang-format on C++ files to apply :ref:`automatic code formatting
+   <styleguide_formatting>`. In addition, you may want to configure your text
+   editor to automatically run clang-format when saving C++ files.
```

Comment:
I think there should be a note here saying that clang-format installation is a separate process from actually running the post-commit hook. As someone who was new to this, this wasn't clear to me until I tried running the `install-commit-hooks.sh` script and saw the error that git-clang-format wasn't installed on my machine

Reply:
Thanks for the suggestion! I just added that (as well as a note in styleguide.rst)